src/onepasswordconnectsdk/client.py

```diff
@@ -184,11 +184,12 @@ def get_item_by_title(self, title: str, vault_id: str):
         item_summary = self.deserialize(response.content, "list[SummaryItem]")[0]
         return self.get_item_by_id(item_summary.id, vault_id)
 
-    def get_items(self, vault_id: str):
+    def get_items(self, vault_id: str, filter_query=None):
         """Returns a list of item summaries for the specified vault
 
         Args:
             vault_id (str): The id of the vault in which to get the items from
+            filter_query (str): A optional query statement. `title eq foo.bar`
```

Comment:
I'd put exactly the same examples  as on dev portal documentation 
```
title eq "Example Item" or tag eq "banking"
```

Reply:
addressed in [this commit](https://github.com/1Password/connect-sdk-python/pull/62/commits/feeeb87beb6781a1b968e7bb10506d2c3ba3b142)